CHANGELOG.md

```diff
@@ -1,3 +1,31 @@
+# dbt_facebook_ads_source v0.9.0
+
+[PR #44](https://github.com/fivetran/dbt_facebook_ads_source/pull/44) introduces the following updates:
+
+## Schema Updates
+
+**9 total changes • 0 possible breaking changes**
+| **Data Model** | **Change type** | **Old name** | **New name** | **Notes** |
+| ---------------- | --------------- | ------------ | ------------ | --------- |
+| stg_facebook_ads__demographics_country | New Model |   |   |  Uses `demographics_country` source table  |
```

Comment:
Request to add links to the DAG for the **Data Model** column

Reply:
Added